impl/src/main/java/org/pentaho/di/trans/dataservice/resolvers/TransientResolver.java

```diff
@@ -122,6 +124,10 @@ private DataServiceMeta createDataServiceMeta( String dataServiceName ) {
         local = true;
         stepName = stepName.replace( LOCAL, "" );
       }
+      if ( stepName.startsWith( STREAMING ) ) {
```

Comment:
Is this working after the LOCAL above because of this order of this in [DataServiceMetaFactory](https://github.com/pentaho/pdi-dataservice-server-plugin/pull/279/files#diff-5980933c914e857e00500d96cf032427)?:
`buildTransient( fullFileName, local + stream + step.getName(), rowLimit );`

Shouldn't we add a third case in the DataServiceMetaFactory that adds the LOCAL and STREAMING, and then deal with it in here, something like a third if:
`if ( stepname.startsWith( STREAMING_LOCAL ) ) {`

This way the code doesn't brake, if someone changes the line order for this if here, or the step name in DataServiceMetaFactory.